classnames/classnames.d.ts

```diff
@@ -6,7 +6,7 @@
 declare type ClassValue = string | number | ClassDictionary | ClassArray;
 
 interface ClassDictionary {
-	[id: string]: boolean;
+	[id: string]: any;
```

Comment:
Sorry, the reason it's not working for me is because I am using TS strict-mode :)